bin/plugin/commands/common.js

```diff
@@ -68,5 +82,26 @@ exports.getModuleData = async ( modulesDir ) => {
 
 			return moduleData;
 		} )
-		.filter( ( moduleData ) => moduleData.name && moduleData.description );
+		.filter( ( moduleData ) => moduleData.name && moduleData.description )
+		.sort( ( firstModule, secondModule ) => {
+			// Not the same focus group.
+			if ( firstModule.focus !== secondModule.focus ) {
+				return FOCUS_AREAS[ firstModule.focus ] >
+					FOCUS_AREAS[ secondModule.focus ]
+					? 1
+					: -1;
+			}
+
+			const bothModulesAreExperimental =
+				firstModule.experimental && secondModule.experimental;
+			if (
+				! bothModulesAreExperimental &&
+				( firstModule.experimental || secondModule.experimental )
+			) {
```

Comment:
This looks overly complicated and potentially slightly off? We only need to compare based on experimental state if only one of the two modules to compare is experimental, otherwise this property isn't relevant as they are equal in that sense (similar to how you do it for the focus areas).
```suggestion
			if ( firstModule.experimental !== secondModule.experimental ) {
```

Reply:
Woops looks like I over complicated things on that one, agree just checking on different values would be enough at this level, we might need some additional level inside of the block but that is a different story. :+1:

Reply:
Actually looks like the body of the code should work as it is :)